modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -3790,6 +3790,11 @@ private void _assertSXPBlueprint1() throws Exception {
 				"TESTSXPBLUEPRINT1");
 
 		Assert.assertNotNull(sxpBlueprint);
+		Assert.assertFalse(
```

Comment:
Assert if the content was replaced by correct string.

Reply:
Or actually you can do the same as:

Assert.assertFalse(
			pageDefinitionString.contains(
				"[$TestObjectDefinition3#Test_Object_Entry_1$]"));

Reply:
Done!